### ISSUE TITLE
emo-bon/governance-data comments

### ISSUE THREAD

**Issue by cymon** [emo-bon/governance-data#24]
a) There's a column in ./observatories.cvs name "contect person" should be "contact_person" with underscore
b) "contact person email" should be "contact_person_email" with underscores
c) Column that have the same data but are in different sheets should have the same column names:
e.g EMOBON_observatory_name in observatories.csv
and operating_observatory_id in organisation.csv
d) consider combining observatories.csv and organistations.csv to same on duplicating data.
e) there a quite a few cells with blanks strings in them " "... rather than being NULL/NaN

**Comment by kmexter** [comment]
a) and b) I changed as they are typoes, but I will also need to pass this on to Bram so they can check if this breaks anything in the harvesting/qc code
c) I agree, but before changing one or the other I need to check with Bram on that also
d) not sure why ioulia wanted these two separate, but agree could be combined - need to check first where this is used before doing that

**Comment by kmexter** [comment]
I will change A and B and then tell bram

**Comment by isanti** [comment]
Yes, fine by me to combine observatories.csv and organistations.csv
This will also solve c) as we will delete redundant columns
@isanti to combine observatories.csv and organistations.csv

We could also combine the logsheets.csv with the above. Will this affect anyone's code? Waiting for confirmation on this

@isanti add NaN to the empty columns

**Comment by isanti** [comment]
A! I know why we have two files. It's because we have more operators than observatories. Belgium has formed one observatory (BPNS) but 4 operators collaborate for it. We need to include and acknowledge them all. Therefore, the row in organizations.csv are more than the row in observatories.csv

Then we have contact_person in observatories.csv and OpCo_representative in organizations.csv. These are different roles and can be different people.

I did change columns titles to have EMOBON_observatory_id and country in both files. And added NaN for empty values. But other than that the files can stay like this,
